transformation/README.md

```diff
@@ -0,0 +1,11 @@
+# Quadkey
+
+On this module we use Quadints, our own version of [Quadkeys](https://wiki.openstreetmap.org/wiki/QuadTiles) stored in a INT64. Quadints have multiple advantages as faster comparision, consistent storage or better indexing.
+
+We also provide the functions necessary to parse between Quadints and Quadkeys.
+
+## Folder structure
+
+bq/      - Files necessary to generate BigQuery UDFs.  
+sf/      - Files necessary to generate Snowflake UDFs.  
+library/ - Base code used to generate the final JS library.
```

Comment:
This is all copy pasted from quadkey. I recommented to have copied the module, "skel" which has the purpose of been copied rather than a custom module. And then with your editor, mine is vscode, replacing the word "skel" by "mymodule" and "SKEL" by "MYMODULE". This is a way to save some time.